include/CLI/App.hpp

```diff
@@ -2390,8 +2404,8 @@ class App {
     ///
     /// If this has more than one dot.separated.name, go into the subcommand matching it
     /// Returns true if it managed to find the option, if false you'll need to remove the arg manually.
-    void _parse_config(const std::vector<ConfigItem> &args) {
-        for(const ConfigItem &item : args) {
+    void _parse_config(std::vector<ConfigItem> &args) {
+        for(ConfigItem item : args) {
```

Comment:
Why this change?

Reply:
recommended by clang-tidy or some other static analysis tools I ran

Reply:
I am going to change it back, it might have been a merge conflict or something